server/odc-service/src/main/java/com/oceanbase/odc/metadb/task/JobEntity.java

```diff
@@ -57,6 +57,12 @@ public class JobEntity implements Serializable {
     @Column(name = "job_type", nullable = false)
     private String jobType;
 
+    @Column(name = "job_parameters_json")
+    private String jobParametersJson;
+
+    @Column(name = "job_properties_json")
+    private String jobPropertiesJson;
```

Comment:
It is recommended not to use the json type to pass framework level parameters

Reply:
framework level parameters will not set in jobPropertiesJson,  jobPropertiesJson only store `JobProperties` which is single job level